src/main/java/frc/robot/Constants.java

```diff
@@ -22,8 +22,8 @@ public static class DriveConstants {
     public static final int kLeftMotorPort2 = 2;
     public static final int kRightMotorPort = 3;
     public static final int kRightMotorPort2 = 4;
-    public static final int intakeMotor = 5;
-    public static final int intakeMotor2 = 6;
+    public static final int kintakeMotorPort = 5;
```

Comment:
Style suggestions: given that the other vars in this file capitalize the word after the `k` prefix, these vars should, as well.

Reply:
Also, generally one would name constants in `ALL_CAPS_WITH_UNDERSCORES` instead of `camelCase`.